net-libs/libtorrent/libtorrent-0.13.6.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit libtool toolchain-funcs
+
+DESCRIPTION="BitTorrent library written in C++ for *nix"
+HOMEPAGE="https://rakshasa.github.io/rtorrent/"
+SRC_URI="http://rtorrent.net/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris"
+IUSE="debug ipv6 ssl test"
+
+RDEPEND="
+	>=dev-libs/libsigc++-2.2.2:2
+	ssl? ( dev-libs/openssl:0= )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	test? ( dev-util/cppunit )"
+
+src_prepare() {
+	elibtoolize
+}
+
+src_configure() {
+	# configure needs bash or script bombs out on some null shift, bug #291229
+	CONFIG_SHELL=${BASH} econf \
+		--disable-dependency-tracking \
+		--enable-aligned \
+		$(use_enable debug) \
+		$(use_enable ipv6) \
+		$(use_enable ssl openssl) \
+		--with-posix-fallocate
+}
```

Comment:
I get a seemingly useless `/usr/lib64/libtorrent.la`, so maybe we want to run `prune_libtool_files` from eutils.eclass?